src/Destructurama.Attributed/Attributed/AttributedDestructuringPolicy.cs

```diff
@@ -56,8 +59,29 @@ private static IEnumerable<PropertyInfo> GetPropertiesRecursive(Type type)
             var unseenProperties = type.GetProperties(BindingFlags.Public | BindingFlags.Instance | BindingFlags.DeclaredOnly)
                 .Where(p => p.CanRead && p.GetMethod.IsPublic && p.GetIndexParameters().Length == 0 && !seenNames.Contains(p.Name));
 
+#if NETSTANDARD2_1_OR_GREATER
+            var metaProp = new List<PropertyInfo>();
+            // find Metadata Class
+            // Take only first Entry, unclear from docs whether it can be specified multiple times, would be nonsense from my perspective
+            var metaDataType = type.GetCustomAttributes<MetadataTypeAttribute>(true).ToList().FirstOrDefault();
```

Comment:
May be potentially breaking so let's introduce `AttributedDestructuringPolicyOptions.RespectMetadataTypeAttribute`.

Reply:
Uncertain, as I did not completly understand the reminder of the code. I have never worked with Expression and things like that. However, it breaks for shure with framework below NETSTANDARD 2.1 as there is no metadatatype attribute support. An application based attribute won't fix this from my perspective. 

What I do not know is, what happens with the propertyinfo after it has been returned and if it matters whether the propertyinfo comes from the metadata class. In other words, I guess if the later code accesses e.g. the value of the property via propertyinfo, than it could be problematic and may not work. In this case  a different strategy is requried and this will have a greater impact to the code.

Reply:
Existing code with `MetadataTypeAttribute` in it should not change generated API. This should be opt-in option.

Reply:
Good point. I'll implement a proposal.